.github/workflows.src/release.tpl.yml

```diff
@@ -12,15 +12,13 @@ jobs:
     steps:
     - name: Determine package version
       shell: bash
-      env:
-        BRANCH: "${{ github.event.client_payload.branch }}"
-      run: echo ::set-output name=version::${BRANCH#releases/}
+      run: echo ::set-output name=version::${GITHUB_REF#releases/}
       id: whichver
 
     - name: Build
       uses: edgedb/edgedb-pkg/integration/linux/build/<< tgt.name >>@master
       env:
-        SRC_REF: "${{ github.event.client_payload.branch }}"
+        SRC_REF: $GITHUB_REF
```

Comment:
Not sure if this will work? If not, we need to update edgedb-pkg to use GITHUB_REF instead.

Reply:
The correct syntax is "${{ env.GITHUB_REF }}", I think.

Reply:
Also, you can normalize the branch name from `GITHUB_REF` in the "Determine package version" step, just add another output